src/eko/evolution_operator/__init__.py

```diff
@@ -245,7 +246,12 @@ class Operator:
             cut to the upper limit in the mellin inversion
     """
 
-    def __init__(self, config, managers, nf, q2_from, q2_to, mellin_cut=5e-2):
+    operator_type = "Evolution"
```

Comment:
I see you use it for logging - maybe `log_label` ? in principle this would not be needed since the logger is scoped by `__name__` and hence knows who is logging ... maybe we should change our default log format in `ekomark` and use the more verbose `logging.Formatter("%(asctime)s %(name)s/%(levelname)s: %(message)s")` as [here](https://github.com/N3PDF/pineko/blob/fkutils-move/src/pineko/theory.py#L248) - of course at the price of spam